domain-model/src/main/java/uk/gov/hmcts/cmc/domain/exceptions/BadRequestException.java

```diff
@@ -6,4 +6,8 @@ public BadRequestException(String message, Exception ex) {
         super(message, ex);
     }
 
+    public BadRequestException(String message) {
```

Comment:
Is this mapped to 400 HTTP response anywhere? Isn't there anything available from Spring that we could perhaps reuse?